FindFreetype.cmake

```diff
@@ -1,19 +1,21 @@
+cmake_minimum_required(VERSION 2.8)
 
+foreach(libname ${CONAN_LIBS})
+    find_library(${libname}_LIBRARY NAMES ${libname} PATHS ${CONAN_LIB_DIRS} NO_CMAKE_FIND_ROOT_PATH)
```

Comment:
It should collect only freetype dependencies in FREETYPE_LIBRARY.

Reply:
conanbuildinfo.cmake only list freetype and libm as dependencies, so libpng, zlib and bzip2 are separated. I didn't find a smart way to solve it.